### DISCUSSION TITLE
The i18n branch is out of date?

### DISCUSSION THREAD

**Discussion by zsxeee** [hoppscotch/hoppscotch#2040] [Q&A] [answered]
The latest version is 2.1.0, but when I check out the i18n branch, all translation files are still in 2.0 and not synced with the live version.
Where should I start? check out the main branch then merge back to the i18n branch, or just wait for the i18n branch update?

**Comment by liyasthomas** [comment, marked as answer]
I've updated the `i18n` branch with the latest commits. You can start contributing to it now. Thank you for noticing.
